e2e-tests/portal/cypress/e2e/journeys/payment-report-officer/utilisation-report-upload/can-upload-monthly-utilisation-report.spec.js

```diff
@@ -95,7 +96,7 @@ context('Monthly utilisation report upload', () => {
 
       utilisationReportUpload.checkReportTitle().should('exist');
       utilisationReportUpload.validationErrorTable().should('exist');
-      utilisationReportUpload.validationErrorTableRows().should('have.length', 7);
+      utilisationReportUpload.validationErrorTableRows().should('have.length.above', 0);
```

Comment:
This change (and the other similar changes below) make this test more future-proof. As far as I can see, there is no reason for this test to care necessarily about how many errors appear on the page. Instead, it seems like it only cares that some errors _do_ appear on the page. Happy to hear other opinions if people disagree

Reply:
I'm happy with this change. I considered briefly if we should actually assert on the specific errors which I think is the other way to make this test a bit better but I think that is overkill as we have extensive unit tests covering the message generation and would make it less future proof not more, so I think this is good.

Reply:
Sorry I don't agree - unit tests are great but that's only one piece of the whole feature.

i think we should add specific length and error message checks.

Reply:
Unless it's a quick thing, shall we do this in a different ticket/PR @RLavender98 (checking for the exact error messages)?

Happy to resolve if we can add a specific length check